spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitTests.java

```diff
@@ -60,6 +61,8 @@
 	@Override
 	@Test
 	public void sampleConfiguration() {
+		String property = System.getProperty("java.version");
+		System.out.println(property);
```

Comment:
I think it is an overhead.
I believe you that you have had a good testing around there.
Although I'll recheck test tasks with IO during merge.